### ISSUE TITLE
Filling input seems to be broken with :firefox

### ISSUE THREAD

**Issue by metametadata** [igrishaev/etaoin#64]
**Environment:** MacOS 10.13.1, geckodriver 0.19.1.
Reproduced with Firefox 56.0.2 (stable), Firefox 57.0 (Beta), Firefox Nightly 58.0a1 (2017-11-12).

I can't reproduce this bug with `:chrome` and `:phantom`.

Steps to reproduce:

1) Use `:firefox` driver.
2) Try to fill the input:

```clj
(e/fill d {:id :username} "foobar")
```

**Expected:** no errors.

**Actual:**

```
ERROR in (my-test) (support.clj:201)
Uncaught exception, not in assertion.
expected: nil
  actual: clojure.lang.ExceptionInfo: throw+: {:response {:value {:error "invalid argument", :message "Missing 'text' parameter", :stacktrace "stack backtrace:\n   0:        0x101c90804 - backtrace::backtrace::trace::hf4fb9cb16c5cd3d5\n   1:        0x101c90bdf - backtrace::capture::Backtrace::new::h109fc438e1b250fa\n   2:        0x1019c824c - webdriver::error::WebDriverError::new::h55f28d494fb52985\n   3:        0x1019bb1a2 - _$LT$webdriver..command..SendKeysParameters$u20$as$u20$webdriver..command..Parameters$GT$::from_json::hcec9d3aa53ea79d9\n   4:        0x1018dd477 - _$LT$webdriver..command..WebDriverMessage$LT$U$GT$$GT$::from_http::hf39003ef8bb4fef9\n   5:        0x1018e0179 - _$LT$webdriver..httpapi..WebDriverHttpApi$LT$U$GT$$GT$::decode_request::h22eca87d078a6dc4\n   6:        0x10192ac07 - _$LT$webdriver..server..HttpHandler$LT$U$GT$$u20$as$u20$hyper..server..Handler$GT$::handle::h2b535e3cc76d3f78\n   7:        0x101844078 - _$LT$hyper..server..Worker$LT$H$GT$$GT$::keep_alive_loop::h45f7f92b91973f2b\n   8:        0x101844f21 - _$LT$hyper..server..Worker$LT$H$GT$$GT$::handle_connection::h5ee65c07c0fed43f\n   9:        0x1018fa1d7 - hyper::server::handle::_$u7b$$u7b$closure$u7d$$u7d$::h3fe31a4cf8d7cf04\n  10:        0x1018fa75d - hyper::server::listener::spawn_with::_$u7b$$u7b$closure$u7d$$u7d$::hf33d4e4ab2936cf9\n  11:        0x1018479f7 - std::sys_common::backtrace::__rust_begin_short_backtrace::h024931f7d16d4d84\n  12:        0x101858cbd - std::thread::Builder::spawn::_$u7b$$u7b$closure$u7d$$u7d$::_$u7b$$u7b$closure$u7d$$u7d$::h4f89971b92f1c7a8\n  13:        0x101808ca7 - _$LT$std..panic..AssertUnwindSafe$LT$F$GT$$u20$as$u20$core..ops..function..FnOnce$LT$$LP$$RP$$GT$$GT$::call_once::h0f3b12cd61063ffe\n  14:        0x10185994f - std::panicking::try::do_call::he35b3e04abf19639\n  15:        0x101f9cc8c - __rust_maybe_catch_panic\n  16:        0x1018592fc - std::panicking::try::h6891a1b8bc2b2efc\n  17:        0x1018566d2 - std::panic::catch_unwind::hf8158f85675cb440\n  18:        0x1018586cc - std::thread::Builder::spawn::_$u7b$$u7b$closure$u7d$$u7d$::h7ca9b1c3c82280dd\n  19:        0x1018c5da1 - _$LT$F$u20$as$u20$alloc..boxed..FnBox$LT$A$GT$$GT$::call_box::h5a6b04902633fc56\n  20:        0x101f98e7b - std::sys::imp::thread::Thread::new::thread_start::h65773f24735ca5ff\n  21:     0x7fff5d50e6c0 - _pthread_body\n  22:     0x7fff5d50e56c - _pthread_start"}}, :path "session/bb454ccc-15ec-c646-89dd-c2958435f8ce/element/7533efe0-0cd1-6e49-a3f2-5cbbeaabf963/value", :payload {:value ["f" "o" "o" "b" "a" "r"]}, :method :post, :type :etaoin/http-error, :port 29983, :host "127.0.0.1", :status 400, :driver {:args ("geckodriver" "--port" 29983), :capabilities {:loggingPrefs {:browser "INFO"}}, :process #object[java.lang.UNIXProcess 0x77c66ac2 "java.lang.UNIXProcess@77c66ac2"], :locator "xpath", :type :firefox, :env nil, :port 29983, :host "127.0.0.1", :url "http://127.0.0.1:29983", :session "bb454ccc-15ec-c646-89dd-c2958435f8ce"}}
 at slingshot.support$stack_trace.invoke (support.clj:201)
    etaoin.client$call.invokeStatic (client.clj:97)
    etaoin.client$call.invoke (client.clj:64)
    etaoin.api$fill_el.invokeStatic (api.clj:1822)
    etaoin.api$fill_el.doInvoke (api.clj:1819)
    clojure.lang.RestFn.invoke (RestFn.java:445)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.RestFn.applyTo (RestFn.java:132)
    clojure.core$apply.invokeStatic (core.clj:663)
    clojure.core$apply.invoke (core.clj:652)
    etaoin.api$fill.invokeStatic (api.clj:1835)
    etaoin.api$fill.doInvoke (api.clj:1826)
    clojure.lang.RestFn.invoke (RestFn.java:445)
    e2e.auth$fn__12204$fn__12205.invoke (auth.clj:25)
...
```

**Comment by igrishaev** [comment]
@metametadata thank you for pointing that out. That's exactly what I discuss in my blog: http://grishaev.me/en/mozilla

At the moment, I'm thinking on how to fix that keeping backward capability. I mean to not break the API version that we have right now.

**Comment by igrishaev** [comment]
@metametadata it has been fixed in the latest 0.1.9-SNAPSHOT.

**Comment by metametadata** [comment]
Can confirm. Thank you!
